docs/developer-guide/working-with-rules.md

```diff
@@ -261,6 +261,8 @@ Once you have an instance of `SourceCode`, you can use the methods on it to work
 * `getTokensAfter(nodeOrToken, count)` - returns `count` tokens after the given node or token.
 * `getTokensBefore(nodeOrToken, count)` - returns `count` tokens before the given node or token.
 * `getTokensBetween(node1, node2)` - returns the tokens between two nodes.
+* `getTokenOrCommentAfter(nodeOrToken)` - returns the first token or comment after the given node or token.
```

Comment:
I'm actually going to remove these changes since the [TokenStore rewrite](https://github.com/eslint/eslint/pull/7936) updates this documentation.